modules/greenbidsRtdProvider.js

```diff
@@ -96,7 +97,7 @@ function updateAdUnitsBasedOnResponse(adUnits, responseAdUnits, greenbidsId) {
       if (isFilteringForced) {
         adUnit.bids = [];
         logInfo('Greenbids Rtd: filtering flag detected, forcing filtering of Rtd module.');
-      } else if (!matchingAdUnit.isExploration) {
+      } else if (!matchingAdUnit.isExploration & !isFilteringDisabled) {
         removeFalseBidders(adUnit, matchingAdUnit);
       }
```

Comment:
suggestion: may I ask to rewrite the condition to make it more straight forward?
```suggestion
if (matchingAdUnit.isExploration || isFilteringDisabled) return;  // Or you may log something
if (isFilteringForced) {
   ...
} else {
   removeFalseBidders(...);
}